data/infra/resources/remote_file.yaml

```
@@ -9,7 +9,7 @@ resource_description_list:
 - markdown: Use the **remote_file** resource to transfer a file from a remote location
     using file specificity. This resource is similar to the **file** resource.
 - note:
-    markdown: Fetching files from the `files/` directory in a cookbook should be done
+  - markdown: Fetching files from the `files/` directory in a cookbook should be done
```

The note text is missing.
```suggestion
  markdown: Fetching files from the `files/` directory in a cookbook should be done
```

---

This is how we're generating this one currently. Should this just be a single hash?

---

Yep. There's a few we need to fix.